account_move_line_purchase_info/__manifest__.py

```diff
@@ -1,12 +1,12 @@
-# Copyright 2017 Eficent Business and IT Consulting Services S.L.
-#           (www.eficent.com)
+# Copyright 2019 ForgeFlow S.L.
+#   (https://www.forgeflow.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Account Move Line Purchase Info",
     "summary": "Introduces the purchase order line to the journal items",
-    "version": "12.0.2.0.0",
-    "author": "Eficent, " "Odoo Community Association (OCA)",
+    "version": "13.0.1.0.0",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA)",
```

Comment:
can you remove the double quotes in the middle?

Reply:
Done :+1: